pom.xml

```diff
@@ -31,6 +31,8 @@
     <org.jsoup.version>1.7.1</org.jsoup.version>
     <pde.core.version>3.11.1</pde.core.version>
     <sat.version>0.2.0</sat.version>
+    <commonmark.version>0.9.0</commonmark.version>
+    <org.apache.maven.plugins.version>2.12.4</org.apache.maven.plugins.version>
```

Comment:
I would expect `maven.surefire.plugin.version` the group is  shared by many more bindings with different versioning